package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -509,6 +509,10 @@ dhcp_boot_add() {
 	dhcp_option_add "$cfg" "$networkid" "$force"
 }
 
+eval_ipcalc() {
+	local output="$(ipcalc.sh "$@")" || return
```

Comment:
I needed the `local output` on a separate line to the `output=` for this to function as expected.